src/styles/index.scss

```diff
@@ -1,3 +1,71 @@
+html {
+  height: 100%;
+}
+
 body {
   margin: 0;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  position: relative;
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+
+  &__minutes {
+    position: absolute;
+    left: calc(50% - 1.5vmin);
+    top: calc(50% - 20vmin);
+    content: '';
+    display: block;
+    width: 3vmin;
+    height: 20vmin;
+    background-color: #0700ff;
+    animation: nev 3600s infinite steps(60);
+    transform-origin: bottom center;
+  }
+
+  &__minutes--speed-up {
+    animation-duration: 600s;
+  }
+
+  &__sekundes {
+    position: absolute;
+    top: calc(50% - 38vmin);
+    left: calc(50% - 1vmin);
+    content: '';
+    display: block;
+    width: 1.5vmin;
+    height: 38vmin;
+    background-color: #2c8000;
+    transform: rotate(1deg);
+    animation: nev 60s infinite;
+    transform-origin: bottom center;
+    animation-timing-function: linear;
+  }
+
+  &__sekundes--speed-up {
+    animation-duration: 10s;
+  }
+
+  &__center {
+    position: absolute;
+    top: calc(50% - 2.5vmin);
+    left: calc(50% - 2.5vmin);
+    content: '';
```

Comment:
The `content: '';` property is used with the `::before` and `::after` pseudo-elements. It's not necessary here and can be removed.